m5stack-atom-echo/m5stack-atom-echo.yaml

```diff
@@ -279,6 +279,32 @@ switch:
       - switch.turn_off: timer_ringing
 
 select:
+  - platform: template
+    name: "Active wake word"
+    icon: "mdi:bullhorn"
+    entity_category: config
+    options:
```

Comment:
? should add a "DISABLED" option that is selected when "Wake word engine location" is "In home assistant"?